modules/claims_api/spec/fixtures/v2/veterans/power_of_attorney/2122/valid.json

```diff
@@ -2,7 +2,12 @@
   "data": {
     "attributes": {
       "serviceOrganization": {
-        "poaCode": "083"
+        "poaCode": "083",
+        "registrationNumber": "67890"
+      },
+      "signatures": {
```

Comment:
I think an older comment asked about this. I thought we were removing the signatures param entirely for POA V2.